platform/darwin/src/MGLStyle.h

```diff
@@ -259,17 +259,19 @@ static const NSInteger MGLStyleDefaultVersion = 9;
 - (void)removeStyleClass:(NSString *)styleClass;
 
 /**
- Sets an image to the sprite atlas with a unique identifier.
+ Sets the name and image to the style.
+ The image can be used by setting the MGLSymbolStyleLayer.iconImage
+ property to the name passed in here
```

Comment:
This method can be used to both add a new image or override an existing one, so we should make that clear. Also, when [jazzy](https://github.com/realm/jazzy/) generates the HTML documentation, it automatically links symbols enclosed in backticks, to help readers navigate the multi-page documentation. This wording would be more verbose, but it leaves nothing open to interpretation:

> Adds or overrides an image used by the style’s layers.
> 
> To use an image in a style layer, give it a unique name using this method, then set the `iconImage` property of an `MGLSymbolStyleLayer` object to that name.